docs/adr/0002-use-slf4j-for-logging.md

```diff
@@ -1,31 +1,43 @@
 # Use slf4j together with log4j2 for logging
 
-* Currently JabRef uses apache commons logging 1.2 for logging errors and messages. However, this is not compatible with java 9 and is superseeded by log4j.
+* Up to version 4.1 JabRef uses apache-ommons-logging 1.2 for logging errors and messages. However, this is not compatible with java 9 and is superseded by log4j.
 * SLF4J provides a facade for several logging frameworks, including log4j and supports already java 9
 * Log4j is already defined as dependency and slf4j has already been required by a third party dependency
 
 ## Considered Alternatives
 
 * [Log4j2](https://logging.apache.org/log4j/2.x/)
-* [SLF4J](https://www.slf4j.org/)
-
+* [SLF4J with Log4j2 binding](https://logging.apache.org/log4j/2.x/maven-artifacts.html)
+* [SLF4J with Logback binding](https://logback.qos.ch/)
 
 ## Decision Outcome
+* We chose slf4j with log4j2 binding, because it only requires minimal changes to our logging infrastructure and it is claimed that  
+> Apache Log4j 2 is an upgrade to Log4j that provides significant improvements over its predecessor, Log4j 1.x, and      provides many of the improvements available in Logback while fixing some inherent problems in Logback’s architecture.
+* Furthermore, as slf4j is a facade for several loggers, the underlying implementation can easily be changed in the future
+
+
+## Pros and Cons of the Alternatives 
+
+### Log4j2
+
+* `+` Already defined
```

Comment:
Do you mean "already defined as dependency"? --> Maybe rephrase to "Dependency already exists"